test/e2e/components/vl-element.js

```diff
@@ -95,6 +95,23 @@ class VlElement extends WebElement {
     async scrollIntoView() {
         return this.driver.executeScript('return arguments[0].scrollIntoView()', this);
     }
+
+    async isInViewport() {
+        const bounding = await this.driver.executeScript('return arguments[0].getBoundingClientRect()', this);
+    	const height = await this.driver.executeScript('return (window.innerHeight || document.documentElement.clientHeight)');
+    	const width = await this.driver.executeScript('return (window.innerWidth || document.documentElement.clientWidth)');
+    	const outOfViewport = {
+   			top: bounding.top < 0,
+   			left: bounding.left < 0,
+   			bottom: bounding.bottom > height,
+   			right: bounding.right > width
+    	};
+    	return outOfViewport.top && outOfViewport.left && outOfViewport.bottom && outOfViewport.right;
```

Comment:
Klopt deze logica wel? Je zegt hier als de 4 zijden van de rectangle uit de viewport zijn dan is het element inViewport?

Reply:
good call, ik ging er van uit dat dit gewoon geport was vanuit modal?

Reply:
daar staat
return outOfViewport.top || outOfViewport.left || outOfViewport.bottom || outOfViewport.right;

Reply:
dus nie gewoon gemoved he @coemans

Reply:
Klopt, omgedraaid, maar `!` vergeten en testen waren toevallig groen. Wordt gefixt in de volgende release. Kudo's voor @Pieter-Beckers.

Reply:
zoude die vorige release nie weg doen dan?

Reply:
Prima, zie #60 en zal opnieuw een 5.0.4 release maken.